26-register_registration/creator_navigates_to_description_tab.feature

```diff
@@ -90,3 +90,36 @@ Feature: Creator navigates to Description tab
     Given Creator begins Wizard registration and navigates to Description tab
     When the user opens the Project wizard Dialog for creating a new Project
     Then User opens the Project Wizard to register a new Project
+
+  @test
+  Scenario: Creator adds funding
+    Given Creator begins Wizard registration and navigates to Description tab
+    When they uses the option to add funding
+    Then they see an option to add a funding source
```

Comment:
Then they may add an additional funding source (to the funding? Or is this an "additional funding")
And they may cancel (the current operation?)

Reply:
```suggestion
    Then they can select a funding source
```

Reply:
Is 'Select a funding source' clearer?

Reply:
I think so, I'm trying to see this without the FE in front of me :D